recipes-bsp/imx-vpu/imx-vpu_5.4.39.3.bb

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2013-2018 O.S. Systems Software LTDA.
+# Copyright (C) 2013-2016 Freescale Semiconductor
+# Copyright 2017-2020 NXP
+
+DESCRIPTION = "Freescale VPU library"
+LICENSE = "Proprietary"
+LIC_FILES_CHKSUM = "file://COPYING;md5=228c72f2a91452b8a03c4cab30f30ef9"
+
+PROVIDES = "virtual/imxvpu"
+RPROVIDES_${PN} = "virtual/imxvpu"
+
+PE = "1"
+
+SRC_URI = "${FSL_MIRROR}/${BP}.bin;fsl-eula=true"
+SRC_URI[md5sum] = "6d6302189a6704874375afe62a65def0"
+SRC_URI[sha256sum] = "87cb799a57df654db29403cb74a75ca5185a1517022d3a4a16b8d69056c36127"
```

Comment:
@thochstein can you get the changelog for us?

Reply:
```
$ git log --oneline --no-decorate 3e47ffe..d8f988d
d8f988d MA-14108 Remove ION_FLAG_CACHED flag for cnm vpu buffer
```